README.md

```diff
@@ -377,7 +377,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Pocket](https://getpocket.com/developer/) | Bookmarking service | `OAuth` | Yes | Unknown |
 | [PrexView](https://prexview.com) | Data from XML or JSON to PDF, HTML or Image | `apiKey` | Yes | Unknown |
 | [Restpack](https://restpack.io/) | Provides screenshot, HTML to PDF and content extraction APIs | `apiKey` | Yes | Unknown |
-| [Salesfly PDF](https://salesflypdf.com) | HTML/URL to PDF | `apiKey` | Yes | Yes |
+| [Salesfly PDF](https://developers.salesfly.com/swift/) | HTML/URL to PDF | `apiKey` | Yes | Yes |
```

Comment:
This API is in private beta.


![image](https://user-images.githubusercontent.com/50463866/123019699-5e4c7400-d3a7-11eb-8e64-981521bf9978.png)

Can you remove, please?

```suggestion
```